test/molecule-role/molecule/secrets/Readme.md

```diff
@@ -0,0 +1,81 @@
+
+Secrets configuring
+
+You need to specify path to the decrypting program by specifying `secret_backend_command`
+command.
+
+All sensitive secrets should follow pattern "ENC[keyname]" 
+
+```
+secret_backend_command: /etc/stackstate-agent/dummy_secret_feeder.sh
+
+# The StackState api key to associate your Agent's data with your organization.
+api_key: "ENC[api_key]"
+
+```
+
+stackstate-agent will pass following json request to stdin of the decrypting program,
+where `secrets` contains array of the secrets to be returned
+
+```
+{
+  "version": "1.0",
+  "secrets": ["keyname", "api_key"]
+}
+```
+
+stackstate-agent expects, that decrypting program will return json output with
```

Comment:
stackstate-agent expects that **the** decrypting program will return **the** json